Code/Tools/AssetProcessor/native/unittests/FileWatcherUnitTests.cpp

```diff
@@ -6,6 +6,7 @@
  *
  */
 
+#include <native/FileWatcher/FileWatcher.h>
```

Comment:
Just curious: Any reason to include this new header when there's no other change in this test file?

Reply:
Before it was getting FileWatcher.h included because it included ApplicationManagerBase.h which included FileWatcher.h.

Now, ApplicationManagerBase.h includes FileWatcherBase.h, so FileWatcher.h isn't included automatically.

FileWatcherUnitTests.cpp is instantiating a FileWatcher (not base) here, so it needs the include:
https://github.com/o3de/o3de/blob/development/Code/Tools/AssetProcessor/native/unittests/FileWatcherUnitTests.cpp#L22